vscode/src/package.ts

```diff
@@ -0,0 +1,38 @@
+
+export interface ConsolePackage {
+  // The JSON from console does not follow the naming-convention.
+  /* eslint-disable @typescript-eslint/naming-convention */
+  name: string;
+  description: string
+  license: string
+  url: string
+  version: string
+  dependencies: ConsoleDependency[]
+  /* eslint-enable @typescript-eslint/naming-convention */
+}
+
+export interface ConsoleDependency {
+  // The JSON from console does not follow the naming-convention.
+  /* eslint-disable @typescript-eslint/naming-convention */
+  url: string
+  version: string
+  /* eslint-enable @typescript-eslint/naming-convention */
+}
+
+export class Package {
```

Comment:
Given that `Package` and `ConsolePackage` are the same you wouldn't need to copy it from one to the other, but I think it's ok. Makes it probably more consistent with other interactions with the console.

Reply:
I will need this structure later anyway.